PyDSS/pyPostprocessor/PostprocessScripts/AutomatedVoltageUpgrade.py

```diff
@@ -689,6 +708,92 @@ def __init__(self, project, scenario, inputs, dssInstance, dssSolver, dssObjects
     def _get_required_input_fields():
         return AutomatedVoltageUpgrade.REQUIRED_INPUT_FIELDS
 
+    def create_final_comparison(self, project_path=None, thermal_dss_file=None):
+        # project_path = project.dss_files_path
+        end_t = time.time()
+        # self.check_voltage_violations_multi_tps(upper_limit=self.upper_limit,
+        #                                         lower_limit=self.lower_limit)
+        # if self.config["create_topology_plots"]:
+        #     self.plot_violations()

```

Comment:
Should these lines be uncommented? Otherwise deleted?

Reply:
deleted. called in another location already